reccmp/isledecomp/formats/detect.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
```

Comment:
We could maybe merge this file into `__init__.py` if it is the main entrypoint for binary images